tox.ini

```diff
@@ -170,6 +170,8 @@ disable=
     # method.
     W0221,
     # unecessary "else" after "return" (R1705)
+    W0632,
```

Comment:
I don't believe this needs to be disabled. There is an unbalance in the function return values and the variables used to unpack it, which needs to be fixed.

Reply:
I have rebased by dropped the committee. Thanks for pointing it out.

Reply:
Problem fixed in bdc3cd1. Thanks for the help.